README.md

```diff
@@ -31,6 +29,42 @@ To run this application, you need to first clone the getting-started-with-the-do
 5. Clean and build the application.
 6. Run the application.
 
+## Features and Benefits
+
+## Precision
+
+The .NET MAUI Rating control supports precise ratings to handle full, [half](https://help.syncfusion.com/maui/rating/precision-mode#half?utm_source=github&utm_medium=listing&utm_campaign=maui-rating-github-samples), or [exact](https://help.syncfusion.com/maui/rating/precision-mode#exact?utm_source=github&utm_medium=listing&utm_campaign=maui-rating-github-samples) values.
+
+## Rating shapes
+
+The Rating control in .NET MAUI allows the use of both predefined shapes and custom shapes.
```

Comment:
Use as below
The Rating control in .NET MAUI allows the use of both predefined shapes and [custom shapes](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.RatingShape.html#Syncfusion_Maui_Inputs_RatingShape_Custom?utm_source=github&utm_medium=listing&utm_campaign=maui-rating-github-samples).